src/scss/partials/_nav.scss

```diff
@@ -241,9 +241,7 @@ $nav-popover-padding          : spacing(x2);
 // Navigation Trigger
 // This is the checkbox that controls the menu active state without JS via :checked
 .c-nav-trigger {
-    position: absolute;
-    top: -100px;
-    left: -100px;
+    display: none;
```

Comment:
Would be good to double check if this affects accessibility in some screen readers.

Most articles I’ve seen seem to advise not to display none screen reader components…